articles/clients/grant-types/client-grant-types.md

```diff
@@ -0,0 +1,124 @@
+---
+  description: Using the Grant Types property on Clients
+  toc: true
+---
+
+# Client Grant Types
+
+Depending on your use case, Auth0 provides many authorization flows that suit your needs. As such, you'll need to track the type of flow you're using for a given client. As such, Auth0 includes a `grant_type` property on each Client for this purpose.
```

Comment:
Recommended instead: 
Depending on your use case, Auth0 provides many authentication and authorization flows that suit your needs, and you may wish to limit the use of certain grant types for a particular client. Auth0 includes a `grant_type` property on each Client for this purpose.